doc/source/feature_comparison.rst

```diff
@@ -20,111 +20,260 @@
-|Improved Handling of Corrupted Tables                   | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|Ability to Kill Idle Transactions                       | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|START TRANSACTION WITH CONSISTENT SNAPSHOT improvements | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-
-
-+---------------------------------------+----------------------+--------------+
-|Running Database as a Service          |Percona Server 5.7.10 |MySQL 5.7.10  |
-+=======================================+======================+==============+
-|Special Utility User                   | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
-|Expanded Program Option Modifiers      | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
-|Enforcing the Specific Storage Engine  | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Features
+      - Percona Server 5.7.24
+      - MySQL 5.7.24
+    * - Open source
+      - Yes
+      - Yes
+    * - ACID Compliance
+      - Yes
+      - Yes
+    * - Multi-Version Concurrency Control
+      - Yes
+      - Yes
+    * - Row-Level Locking
+      - Yes
+      - Yes
+    * - Automatic Crash Recovery
+      - Yes
+      - Yes
+    * - Table Partitioning
+      - Yes
+      - Yes
+    * - Views
+      - Yes
+      - Yes
+    * - Subqueries
+      - Yes
+      - Yes
+    * - Triggers
+      - Yes
+      - Yes
+    * - Stored Procedures
+      - Yes
+      - Yes
+    * - Foreign Keys
+      - Yes
+      - Yes
+    * - GTID Replication
+      - Yes
+      - Yes
+    * - Group Replication
+      - Yes
+      - Yes
+    * - MyRocks Storage Engine
+      - Yes
+      - No
+    * - TokuDB Storage Engine
+      - Yes
+      - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Extra Features for Developers
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - NoSQL Socket-Level Interface
+     - Yes
+     - Yes
+   * - X API Support
+     - Yes
+     - Yes
+   * - InnoDB Full-Text Search Improvements
+     - Yes
+     - No
+   * - Extra Hash/Digest Functions
+     - Yes
+     - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Instrumentation and Troubleshooting Features
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - INFORMATION_SCHEMA Tables
+     - 93
+     - 61
+   * - Global Performance and Status Counters
+     - 789
```

Comment:
This was 389 in the original table? And 789 looks too much to me, are you sure?

Reply:
According to the PS Feature Comparison spreadsheet made during 8.0 GA (Tyler)
Global Performance and Status Counters - 789

Reply:
Similarly here, I'm getting 432 on the 5.7 trunk...

Reply:
changed to 432